owners/test/index.test.js

```diff
@@ -82,11 +81,27 @@ describe('owners bot', () => {
       .stub(OwnersParser.prototype, 'parseAllOwnersRules')
       .returns({result: ownersRules, errors: []});
 
-    probot = new Probot({});
-    app = probot.load(owners);
-
-    // just return a test token
-    app.app = () => 'test';
+    probot = new Probot({
+      // These are fake values (but a valid generated key) required for Probot
+      // to execute its auth flow.
+      id: 123,
+      cert: `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Can we avoid Probot authentication in our tests?

Reply:
Sorta; done